src/shopilingus/ProductTypes.java

```diff
@@ -0,0 +1,8 @@
+package shopilingus;
+
+public enum ProductTypes {
```

Comment:
you should use it in Product class to keep types as a Set<ProductTypes> instead of list of strings

Reply:
rename `ProductTypes` -> `ProductType`

Reply:
So, in Product class should I import ProductType class?